Cpp/AlarmClock/src/AlarmClock.lf

```diff
@@ -1,40 +1,30 @@
-/*
-* This is a minimal example of an alarmclock implemeted using the 
-* features lingua franca supplies.
-* 
-* This is just an extract and simplification from the main project
-* which you can find here: https://github.com/revol-xut/lf-alarm-clock
-* 
-* This file contains the networking implementation it is really just an
-* simple socket application which parses simple http headers and respondes
-* in text/plain
-*
-* @author Tassilo Tanneberer <tassilo.tanneberger@tu-dresden.de>
-*/
-
-target Cpp{
-    cmake-include: "AlarmClock.cmake",
-    keepalive: true
-};
-
-import Network from "./Network.lf";
-import Clock from "./Clock.lf";
+/**
+ * This is a minimal example of an alarmclock implemeted using the features lingua franca supplies.
+ *
+ * This is just an extract and simplification from the main project which you can find here:
+ * https://github.com/revol-xut/lf-alarm-clock
+ *
+ * This file contains the networking implementation it is really just an simple socket application
+ * which parses simple http headers and respondes in text/plain
+ *
+ * @author Tassilo Tanneberer <tassilo.tanneberger@tu-dresden.de>
+ */
+target Cpp {
+  cmake-include: "AlarmClock.cmake",
+  keepalive: true
+}
 
-#import Network.lf;
-#import Clock.lf;
+import Network from "./Network.lf"
+import Clock from "./Clock.lf"
 
+// import Network.lf; import Clock.lf;
```

Comment:
Should we do this?

Reply:
This is hard to avoid if we want to automate line breaking. There are rules about when lines can be merged. For example, lines that start as special symbols that could be acting as bullet points are not merged. If there are multiple lines of comments, and some comments are separated from other comments by a blank line, then they are treated like separate paragraphs and are not merged. We could add similar rules for semicolons, but commented-out code usually is not supposed to be committed anyway, so I'm not sure we should.